core/pom.xml

```diff
@@ -370,5 +370,28 @@
             </activation>
             <!-- not including license-maven-plugin is sufficent to expose default license -->
         </profile>
+        <profile>
+            <id>release</id>
+            <build>
+                <plugins>
+                    <plugin>
+                        <groupId>org.apache.maven.plugins</groupId>
+                        <artifactId>maven-javadoc-plugin</artifactId>
+                        <executions>
+                            <execution>
+                                <id>attach-javadocs</id>
+                                <goals>
+                                    <goal>jar</goal>
+                                </goals>
+                                <configuration>
+                                    <additionalparam>-Xdoclint:none</additionalparam>
```

Comment:
We should not be disabling doclint. We are very careful to fully enable it, except for "missing" docs, in our javac settings.